synth.py

```diff
@@ -45,6 +44,36 @@
         ],
     )
 
+    # We need to parameterize aspects of the client as it varies in different versions.
+    #
+    # In the future once the read and write client are colocated in the same version,
+    # we'll need to loop through through multiple clients.  Perhaps by the time that
+    # happens we'll be on a generator that needs less post-generation modifications.
+    
+    clientinfo = {
+        "file": "big_query_storage_client.py",
+        "type": "storage",
+        "name": "BigQueryStorageClient",
+        "badpkg": "google-cloud-bigquerystorage",
+        "goodpkg": "google-cloud-bigquery-storage",
+    }
+    if version in ["v1beta2","v1"]:
+        clientinfo = {
+            "file": "big_query_read_client.py",
```

Comment:
Am I understanding this correctly that

* `v1beta2` and `v1` only have a read client
* `v1alpha2` only has a write client
* and `v1beta1` only has a read client, but it's called a storage client?

Reply:
That's the gist of it, yes.  I expect once v1alpha2 revs (e.g. to...v2beta1 or similar) we'll have both a read and write client in the same library/version.